libraries/AP_NavEKF2/AP_NavEKF2.h

```diff
@@ -447,6 +447,10 @@ class NavEKF2 {
     const float gndEffectBaroScaler = 4.0f;        // scaler applied to the barometer observation variance when ground effect mode is active
     const uint8_t fusionTimeStep_ms = 10;          // The minimum time interval between covariance predictions and measurement fusions in msec
 
+    // origin set by one of the cores
+    struct Location common_EKF_origin;
+    bool common_origin_valid;
```

Comment:
Convention in the EKF is to set initial values for  class variables in the initialise function.